corehq/apps/cloudcare/views.py

```diff
@@ -441,6 +447,39 @@ def custom_json_handler(obj):
         },
         default=custom_json_handler,
     )
+@cloudcare_api
+def render_form(request, domain, session_id):
+    # get session
+    try:
+        session = EntrySession.objects.get(session_id=session_id)
+    except EntrySession.DoesNotExist:
+        session = None
+
+    response = requests.post(get_url_base() + '/webforms/get-xml/' + session_id)
+    json_response = json.loads(response.text)
+    xmlns = json_response["xmlns"]
+    form_data_xml = json_response["output"]
+
+    _, form_data_json = xml2json(form_data_xml)
+    pretty_questions = readable.get_questions(domain, session.app_id, xmlns)
```

Comment:
it looks like you set the session to None above if it's not found so this line is just going to crash. Simple change would be to change the session initializer to use `get_object_or_404`